openapi/v3/v3-parameters.yaml

```diff
@@ -0,0 +1,150 @@
+openapi: 3.0.3
+info:
+  title: Payments API
+  version: v3
+
+components:
+  parameters:
+    V3AccountID:
+      name: accountID
+      in: path
+      required: true
+      description: The account ID
+      schema:
+        type: string
+
+    V3BankAccountID:
+      name: bankAccountID
+      in: path
+      required: true
+      description: The bank account ID
+      schema:
+        type: string
+
+    V3PaymentID:
+      name: paymentID
+      in: path
+      required: true
+      description: The payment ID
+      schema:
+        type: string
+
+    V3PaymentInitiationID:
+      name: paymentInitiationID
+      in: path
+      required: true
+      description: The payment initiation ID
+      schema:
+        type: string
+
+    V3ConnectorID:
+      name: connectorID
+      in: path
+      required: true
+      description: The connector ID
+      schema:
+        type: string
+
+    V3ScheduleID:
+      name: scheduleID
+      in: path
+      required: true
+      description: The schedule ID
+      schema:
+        type: string
+
+    V3PoolID:
+      name: poolID
+      in: path
+      required: true
+      description: The pool ID
+      schema:
+        type: string
+
+    V3TaskID:
+      name: taskID
+      in: path
+      required: true
+      description: The task ID
+      schema:
+        type: string
+
+    V3Connector:
+      name: connector
+      in: path
+      required: true
+      description: The connector to filter by
```

Comment:
How is this different from connector ID?

Reply:
i'd like to separate them since here it's the name of the connector (ex: mangopay, moneycorp etc...)

Reply:
Ah ok was not clear to me that it was the name of the connector